File: DSA/Madhav_590014266/Day2/Q2.java
public class Main {
    public static void main(String[] args) {
        int[] arr = {3, 7, 9, 2, 5};

        int largest = arr[0];

        for (int i = 1; i < arr.length; i++) {
            if (arr[i] > largest) {
                largest = arr[i];
            }
        }

        System.out.println(largest);
    }
}

File: DSA/Aarush_590017879/Day_2/Q2.c
#include <stdio.h>

int main() {
    int n;

    printf("Enter the number of elements: ");
    scanf("%d", &n);

    int arr[n];

    printf("Enter the elements: ");
    for(int i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }

    int largest = arr[0];

    for(int i = 1; i < n; i++) {
        if(arr[i] > largest) {
            largest = arr[i];
        }
    }
    
    printf("Largest element = %d\n", largest);

    return 0;
}